cclib/parser/gaussianparser.py

```diff
@@ -1278,15 +1278,15 @@ def extract(self, inputfile, line):
                     freqs = [utils.float(f) for f in line[15:].split()]
                     self.vibfreqs.extend(freqs)
 
-                if line[1:15] == "Red. masses --":  # note: matches low-precision block, and
+                if line[1:15] == "Red. masses --":  # note: matches only low-precision block
```

Comment:
I just copied above line, `if line[1:15] == "Frequencies --": # note: matches low-precision block`, and and modified.
I fix comment.